release-notes/3.0/3.0-known-issues.md

```diff
@@ -4,6 +4,14 @@ This document lists known issues for **.NET Core 3.0 GA and beyond releases** wh
 
 ## .NET Core
 
+### .NET Core 3.0.1 SDK (3.0.101 SDK)
+
+- **ASP.NET Core workloads may use older dependencies for dll that are present in the Targeting Pack ([aspnet/Announcements](https://github.com/aspnet/Announcements/issues/398)** 
```

Comment:
Looks like there's a missing closing paren. The one that's there closes the link, then the actual paren in the text is unclosed.

Aslo, should this link text be aspnet/Announcement/398 to line up withou how core-sdk/3331 is linked just below?